react/features/toolbox/components/web/EndConferenceButton.tsx

```diff
@@ -41,6 +41,9 @@ export const EndConferenceButton = (props: Props) => {
 
     const onEndConference = useCallback(() => {
         dispatch(endConference());
+
+        //Gracetech: close the browser
+        window.close();
```

Comment:
does it work on mobile browser?

Reply:
We need it for desktop as well --- and it works if the window is launched from the web app.  If the window is not launched from the web app, the chrome refuses to close because the window didn't originate at this url.
It should work for mobile browser the same, but need more testing.

Reply:
ok, let's try and see